tests/admin_scripts/tests.py

```diff
@@ -1564,6 +1588,27 @@ def handle(self, *args, **options):
         self.assertEqual(out.getvalue(), 'Hello, world!\n')
         self.assertEqual(err.getvalue(), 'Hello, world!\n')
 
+    def test_command_force_color(self):
+        class Command(BaseCommand):
+            requires_system_checks = False
+
+            def handle(self, *args, **options):
+                self.stdout.write('Hello, world!', self.style.ERROR)
+                self.stderr.write('Hello, world!', self.style.ERROR)
+
+        out = StringIO()
+        err = StringIO()
+        command = Command(stdout=out, stderr=err, force_color=True)
+        call_command(command)
+        self.assertEqual(out.getvalue(), '\x1b[31;1mHello, world!\n\x1b[0m')
+        self.assertEqual(err.getvalue(), '\x1b[31;1mHello, world!\n\x1b[0m')
+
+        out = StringIO()
+        err = StringIO()
+        command = Command(stdout=out, stderr=err)
+        with self.assertRaises(CommandError):
+            call_command(command, no_color=True, force_color=True)
```

Comment:
`out` and `err` are unused in the second call. IMO we can simplify this call, e.g.:
```python
with self.assertRaises(CommandError):
    call_command(Command(), no_color=True, force_color=True)
```